src/transform.ts

```diff
@@ -17,6 +17,9 @@ export async function transform(
   resolveId: (id: string) => Promise<string> | string,
   options?: PluginOptions,
 ) {
+  if (!code.includes('import.meta.importGlob') && (!options?.takeover || !code.includes('import.meta.glob')))
+    return
```

Comment:
Well we also take over `import.meta.globEager` and `import.meta.globEagerDefault`. `parseImportGlob` already did some short-circuit logics. How much would these changes improve the performance?

Reply:
> Well we also take over `import.meta.globEager` and `import.meta.globEagerDefault`.

These are also handled by `!options?.takeover || !code.includes('import.meta.glob')`.

>  `parseImportGlob` already did some short-circuit logics. How much would these changes improve the performance?

Ah, didn't see that.

I don't know if it's worth it then. I don't know how negligible it would be compared to the more heavier processing.

Considering that this applies to all modules, even if it's only 50% faster (conservative), it may be worth it. (E.g benchmark comparing `includes()` with RegExp [here](https://stackoverflow.com/questions/5296268/fastest-way-to-check-a-string-contain-another-substring-in-javascript/54303099#54303099).)

Reply:
A side-effect, that may actually have more impact than the performance gains, is that it makes it clear that `vite-plugin-glob` is performant. To avoid erroneous PRs like this one in the future, for example.

Just a thought, I'm not strongly opinionated about this. It's fine to me if we close this PR.